site/static/css/custom.css

```diff
@@ -682,3 +682,7 @@ legend {
 .creditCard:hover {
   background-color: #0d93e2;
 }
+input:focus,
+textarea:focus{
+    outline: none;
+}
```

Comment:
Add a new line here, 
Update a preview link, or gif for the change as it is related to UI